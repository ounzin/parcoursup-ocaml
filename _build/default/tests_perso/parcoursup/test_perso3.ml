open Test_parcoursup_utils2
let session = nouvelle_session ()

let () =  
  ajoute_candidat session ~nom_candidat:"Adam";
  ajoute_candidat session ~nom_candidat:"Lisa";
  ajoute_formation session ~nom_formation:"Université de Monaco" ~capacite:1;
  ajoute_formation session ~nom_formation:"ENSE3 Grenoble" ~capacite:1;
  ajoute_voeu session 
    ~rang_repondeur:(None) 
    ~nom_candidat:"Adam" 
    ~nom_formation:"Université de Monaco";
    ajoute_voeu session 
    ~rang_repondeur:(None)
    ~nom_candidat:"Adam" 
    ~nom_formation:"ENSE3 Grenoble";
    ajoute_voeu session 
    ~rang_repondeur:None
    ~nom_candidat:"Lisa" 
    ~nom_formation:"Université de Monaco";
    ajoute_voeu session 
    ~rang_repondeur:(None) 
    ~nom_candidat:"Lisa" 
    ~nom_formation:"ENSE3 Grenoble";
  ajoute_commission session 
    ~nom_formation:"Université de Monaco" 
    ["Lisa";"Adam"]; 
    ajoute_commission session 
    ~nom_formation:"ENSE3 Grenoble" 
    ["Adam"; "Lisa"]; 
  reunit_commissions session;
  nouveau_jour session;
  affiche_voeux_en_attente session "Adam";
  affiche_propositions_en_attente session "Adam";
  affiche_voeux_en_attente session "Lisa";
  affiche_propositions_en_attente session "Lisa";