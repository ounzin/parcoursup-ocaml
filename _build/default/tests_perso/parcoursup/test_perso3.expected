nouvelle session
ajoute candidat Adam
ajoute candidat Lisa
ajoute formation Université de Monaco (1 places)
ajoute formation ENSE3 Grenoble (1 places)
ajoute voeu de Adam pour Université de Monaco
ajoute voeu de Adam pour ENSE3 Grenoble
ajoute voeu de Lisa pour Université de Monaco
ajoute voeu de Lisa pour ENSE3 Grenoble
commission de classement de Université de Monaco : [Lisa; Adam]
commission de classement de ENSE3 Grenoble : [Adam; Lisa]
réunit commissions
nouveau jour
Voeux en attente de Adam : {Université de Monaco(0)}
Propositions en attente de Adam : {ENSE3 Grenoble}
Voeux en attente de Lisa : {ENSE3 Grenoble(0)}
Propositions en attente de Lisa : {Université de Monaco}
